pkgs/development/misc/avr-gcc-with-avr-libc/default.nix

```diff
@@ -26,8 +26,8 @@ stdenv.mkDerivation {
   
   buildInputs = [ gmp mpfr libmpc zlib ];
   
-  # Make sure we don't strip the libraries in lib/gcc/avr.
-  stripDebugList= [ "bin" "avr/bin" "libexec" ];
+  # Make sure we don't strip the libraries in lib/gcc/avr and in avr/lib.
```

Comment:
Could we have a comment that explains *why* they shouldn't be stripped? Currently it just says the same as the definition below